### PR TITLE
chore(RHINENG-7853): Add OUIA id to create Policy wizard

### PR DESCRIPTION
**Pull request by adonispuente**
This just adds an ouiaId to the policy wizard.

To test, just inspect the page and check the page for pendo-policy-wizard in a ctrl+f. 
Verify that its not there
Then open the policy wizard and see that the id populates and that it points to the entire wizard

### CODE REVIEW

**Review by leSamo — APPROVED** [review]

File: src/components/Policy/__tests__/PolicyWizard.test.tsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ describe('src/components/Policy/PolicyWizard', () => {
         jest.useFakeTimers();
         render(
             <PolicyWizard
+                data-ouia-component-id={ 'pendo-policy-wizard' }
```
Comment (bastilian) [inline src/components/Policy/__tests__/PolicyWizard.test.tsx:26]: Why is this needed?

Comment (leSamo) [reply src/components/Policy/__tests__/PolicyWizard.test.tsx:26]: Pendo is missing selectors for the wizard -- see attached ticket https://issues.redhat.com/browse/RHINENG-7853
